correct heat ordering

Zipfer opened the pull request:
-fix file ordering
  -fix resource ordering
  -Bug #1250776 https://bugs.launchpad.net/fuel/+bug/1250776

mirantis-jenkins commented:
Merged build triggered.

mirantis-jenkins commented:
Merged build started.

mirantis-jenkins commented:
Merged build triggered.

mirantis-jenkins commented:
Merged build finished.

mirantis-jenkins commented:
:+1: Test PASSed.
Refer to this link for build results: http://fuel-jenkins.mirantis.com/job/fuel_syntax_check/648/

Zipfer committed:
correct heat ordering

  -fix file ordering
  -fix resource ordering
  -Bug #1250776 https://bugs.launchpad.net/fuel/+bug/1250776
